### PR TITLE
Misha - the bear

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
image
Adds Misha the bear to icebox HoS office, a snow bear with the same health stats as Sgt. Araneus.
## Why It's Good For The Game
With how easy it is to break into the HoS office on icebox (you don't need any space protection) it never made sense to keep a slow pet in the office. The bear is fast and deals 15 damage, while also resistant to cold.
I think it adds more depth that the HoS tamed a bear on icebox and trained it. Also the bear fits the map.
## Changelog
:cl: grungussuss
add: Added Misha the bear to the HoS office on icebox.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/basic/space_fauna/bear/_bear.dm [hunk 1]
```diff
@@ -82,6 +82,15 @@
 	icon_dead = "snowbear_dead"
 	desc = "It's a polar bear, in space, but not actually in space."
 
+/mob/living/basic/bear/snow/misha
+	name = "Misha"
+	real_name = "Misha"
+	desc = "Tamed and trained by the Head of Security. Only beasts are above deceit"
```
Comment (CRITAWAKETS) [inline code/modules/mob/living/basic/space_fauna/bear/_bear.dm:88]:
```suggestion
	desc = "Tamed and trained by the Head of Security. Only beasts are above deceit."
```
minor spelling mitsake

File: code/game/objects/structures/beds_chairs/bed.dm [hunk 2]
```diff
@@ -272,6 +272,10 @@
 	name = "Cayenne's bed"
 	anchored = TRUE
 
+/obj/structure/bed/dogbed/misha
+	desc = "There is fur all over it, and some blood..."
+	name = "Misha's bed"
+	anchored = TRUE
```
Comment (MrMelbert) [inline code/game/objects/structures/beds_chairs/bed.dm:278]:
```suggestion
	anchored = TRUE

```

**Comment by DaydreamIQ** [comment]
Personally I think Wojtek would be a better name. But yes, good.

**Comment by Sadboysuss** [comment]
> Personally I think Wojtek would be a better name. But yes, good.

I like Warcraft 3 more and HoS isn't Polish

**Comment by Jacquerel** [comment]
every day we get closer to ben10 adding bear riding

**Comment by norsvenska** [comment]
> > Personally I think Wojtek would be a better name. But yes, good.
> 
> I like Warcraft 3 more and HoS isn't Polish

you don't know that, the HoS could be Polish

**Comment by Xkallubar** [comment]
> every day we get closer to ben10 adding bear riding

We already have that

**Comment by Jacquerel** [comment]
> > every day we get closer to ben10 adding bear riding
> 
> We already have that

Every day we get further from the day we already added bear riding

**Comment by atlasle** [comment]
> Every day we get further from the day we already added bear riding

Every 24 hours in SS13, a day passes

**Comment by Loafin34** [comment]
> Personally I think Wojtek would be a better name. But yes, good.

wojtek is in the russian bounty hunter shuttle already

**Pull request merged**